specifyweb/frontend/js_src/lib/components/Forms/DeleteButton.tsx

```diff
@@ -162,6 +163,28 @@ export function DeleteButton<SCHEMA extends AnySchema>({
   );
 }
 
+export function resolveParentViaOtherside(
```

Comment:
No need to export this function. 
It is only used in this file (`DeleteButton.tsx`)